packages/nimbus/src/components/divider/divider.recipe.ts

```diff
@@ -9,28 +9,22 @@ export const dividerRecipe = defineRecipe({
 
   // Base styles applied to all instances of the component
   base: {
-    border: "none",
+    border: "0",
     flexShrink: "0",
-    color: "neutral",
-    borderColor: "colorPalette.6",
+    colorPalette: "neutral",
+    backgroundColor: "colorPalette.6",
   },
 
   // Available variants for customizing the component's appearance
   variants: {
     orientation: {
       horizontal: {
         width: "100%",
-        height: "1px",
-        borderTopWidth: "1px",
-        borderTopStyle: "solid",
-        borderLeftWidth: "0",
```

Comment:
It seemed more straightforward to use a background color to style the divider and not give it both a border and its body, especially since using the border on a 1px unhidden element causes the divider to be 1px off from where it would be expected in the layout